TimeOfFlight/include/TOFUtils.h

```diff
@@ -69,6 +58,12 @@ namespace TOFUtils{
     */
     double getTofFrankFit( std::vector<EVENT::CalorimeterHit*> selectedHits, EVENT::Track* track, double timeResolution);
 
+    /** Get TrackState of the track at the Calorimeter surface. In case of multiple curls take the latest curl (closest to the calorimeter)
+    to get the extrapolated position.
+    */
+    const EVENT::TrackState* geTrackStateAtCalorimeter(EVENT::Track* track);
```

Comment:
```suggestion
    const EVENT::TrackState* getTrackStateAtCalorimeter(EVENT::Track* track);
```
typo I missed previously.